### DISCUSSION TITLE
Home cockpit support - dreaming or??

### DISCUSSION THREAD

**Discussion by Annolago** [flybywiresim/aircraft#4154] [Q&A]
Hi

So I like many others are lusting after building my own home cockpit - however unless I am able to win the Euro jackpot before the rest of you, the chances that I will find that € 250K needed to do so. 
However smaller can also work. Several companies exist that do parts that can make your life as a "simmer" a lot easier. 
So I was looking at Skalarki and noticed that they had an announcement about future support of their products by the FBW crew on their webpage. 
http://www.skalarki-electronics.com/en/blog/flybywire-simulations-cooperation-announcement.html
Great news, if only I had those products - I instead, like many others, have the cheaper solution from VRinsight. The Airbus MCP combo. 
So my question is simply if you plan to extend support of other home cockpit products as part of your long to do list in the future? And if you do are the VRinsight products on your list?

**Comment by tracernz** [comment]
https://github.com/theomessin/jetbridge might be a good path to develop custom solutions.
